src/config/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "accuracy_cutoff": 50,
```

Comment:
the version in the source file should have the former defaults (maybe 80 and 1200).  @theonlydvr @suivo2001 what should these be?